game.py

```diff
@@ -12,16 +12,31 @@
     '-----------'
 ]
 
-
 def snowman(snowman_word):
     """Complete the snowman function
     replace "pass" below with your own code
     It should print 'Congratulations, you win!'
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+    wrong_guesses_list = []
+    while(len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES and not is_word_guessed(snowman_word, correct_letter_guess_statuses)):
```

Comment:
Don't need parens around this expression 

```suggestion
    while len(wrong_guesses_list) < SNOWMAN_MAX_WRONG_GUESSES and not is_word_guessed(snowman_word, correct_letter_guess_statuses):
```

The parentheses indicate an order of operations. If you had an additional part in your conditional, such as an `and` then it would be advisable to use parentheses to indicate how the different expressions should be grouped together. See example below:

```py
if (something == something_else or yet_another_value == something_else) and this == that:
```